### ISSUE TITLE
Keys manager does not extract all keys from ts files

### ISSUE THREAD

**Issue by bilalhaider** [ngneat/transloco-keys-manager#11]
## I'm submitting a...

<pre><code>
[ ] Regression (a behavior that used to work and stopped working in a new release)
[x] Bug report  
[ ] Performance issue
[ ] Feature request
[ ] Documentation issue or request
[ ] Support request
[ ] Other... Please describe:
</code></pre>

## Current behavior

A few translation keys from one particular ts service are not being extracted. This file has other keys as well which are being extracted successfully.

Not extracting:

```typescript

    let progressbarId = this.progressbarsService.createProgressbar(this.translocoService.translate("record.ImportingData"), 0);


      progressbarId = this.progressbarsService.createProgressbar(this.translocoService.translate("LoadingData"), 0); 
```

This part below is interesting because the key `record.ImportStarted` is being extracted and `record.ImportStartedDontNavigateMessage` is not
```typescript
    if (!(options && options.skipSuccessToast)) {
      this.toastsService.showToast({
        severity: "warn",
        summary: this.translocoService.translate("record.ImportStarted"),
        detail: this.translocoService.translate("record.ImportStartedDontNavigateMessage"),
      });
    }
```

I do not have much experience with node cli but I did try to debug the keys-manager and I think the problem is with this regexs.translationCalls based on its matches within this file 

## Environment

<pre><code>
ngular version: 7.3.9

 
For Tooling issues:
- Node version: 13.0.1  
- Platform: Linux  

Others:

</code></pre>

**Comment by shaharkazaz** [comment]
Thanks for the report and the examples, I'll look into it.

**Comment by shaharkazaz** [comment]
@bilalhaider Fixed in v1.2.2
